pack.go

```diff
@@ -215,6 +221,10 @@ func (pb PackBuild) Execute(name, path string) (Image, fmt.Stringer, error) {
 		args = append(args, "--gid", pb.gid)
 	}
 
+	for _, arg := range pb.additionalBuildArgs {
```

Comment:
nit - there's a linter failure here:
`S1011: should replace loop with `args = append(args, pb.additionalBuildArgs...)` (gosimple)`

Reply:
my bad